.github/workflows/config.yaml

```
@@ -0,0 +1,36 @@
+        version: "0.5.0"
+
+    - name: Install dependencies
+      run: |
+        uv venv
+        source .venv/bin/activate
+        uv pip sync pyproject.toml
+
+    - name: Run pyright
+      run: |
+        source .venv/bin/activate
+        pyright . --warnings
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)